src/apps/media/src/app/components/InsightsTable.tsx

```diff
@@ -730,6 +730,9 @@ export const InsightsTable: FC<Props> = ({ files, loading }) => {
               cursor: "pointer",
             },
             border: "none",
+            "& .MuiDataGrid-columnHeaders": {
+              backgroundColor: "grey.100",
```

Comment:
I actually need to confirm this with @zcolah if he prefers this being a theme-level change since I've seen on schema that the table header's color is different.

Reply:
moved to theme change